website/docs/docs/build/sl-getting-started.md

```diff
@@ -4,23 +4,33 @@ title: Get started with MetricFlow
 description: "Learn how to create your first semantic model and metric."
 sidebar_label: Get started with MetricFlow
 tags: [Metrics, Semantic Layer]
+meta:
+  api_name: dbt Semantic Layer API
 ---
 
-This getting started page recommends a workflow to help you get started creating your first metrics. Here are the following steps you'll take:
+This getting started page recommends a workflow to help you create your first metrics. This guide uses the [Jaffle shop example project](https://github.com/dbt-labs/jaffle-sl-template). If you prefer using a sandbox project to follow the guide, you can clone the repo from GitHub. Otherwise, you can create semantic models and metrics for your own dbt project. 
 
-- [Create a semantic model](#create-a-semantic-model)
-- [Create your metrics](#create-your-metrics)
-- [Test and query your metrics](#test-and-query-your-metrics)
+Use this guide to fully experience the power of a universal dbt Semantic Layer. Here are the following steps you'll take:
+
+- [Create a semantic model](#create-a-semantic-model) with MetricFlow
+- [Define metrics](#define-metrics) with MetricFlow
+- [Test and query metrics](#test-and-query-metrics) with MetricFlow
+- [Run a production job](#run-a-production-job) in dbt Cloud
+- [Set up dbt Semantic Layer](#set-up-dbt-semantic-layer) in dbt Cloud
+- [Connect and query API](#connect-and-query-api) with dbt Cloud
 
 ## Prerequisites
 
-- Use the [command line (CLI)](/docs/core/about-the-cli) and have a dbt project and repository set up. 
-  * Note: Support for dbt Cloud and integrations coming soon.
-- Your dbt production environment must be on [dbt Core v1.6](/docs/dbt-versions/core) or higher. Support for the development environment coming soon.
-- Have a dbt project connected to Snowflake or Postgres. 
-  * Note: Support for BigQuery, Databricks, and Redshift coming soon.
 - Have an understanding of key concepts in [MetricFlow](/docs/build/about-metricflow), which powers the revamped dbt Semantic Layer.
-- Recommended &mdash; Install the [MetricFlow CLI](/docs/build/metricflow-cli) to query and test your metrics.
+- Have both your production and development environments running dbt version 1.6 or higher 
+- Use Snowflake, BigQuery, Databricks, or Redshift data platform
+- A successful run in the environment where your Semantic Layer is configured
+- To query with dbt Cloud:
+  * Have a dbt Cloud Team or Enterprise [multi-tenant](/docs/cloud/about-cloud/regions-ip-addresses) deployment, hosted in North America
```

Comment:
Should we add an estimated EMEA/AU timeframe?

Reply:
i think it'll be before coalesce but not positive. i tried avoid committing to it since i was told we don't want to promise timelines to users in the product docs -- but i can put up a (coming soon)